doc/tutorials/ostro-oic-tutorial.md

```diff
@@ -0,0 +1,1030 @@
+# Overview
+
+In this tutorial, we'll show how to develop a simple application using
+Soletta. Although simple, this application will show some important
+features of Soletta, like OIC and HTTP communication, besides GPIO
+interaction.
+Initially, we are going to run the application on desktop [1], but
+later we're going to show how to run it on devices, such as [MinnowBoard
+MAX](http://wiki.minnowboard.org/MinnowBoard_Wiki_Home).
+
+## Step 0
+
+Soletta must be installed on developer's computer. One can use a package
+provide for his or her distro, or [build Soletta](https://github.com/solettaproject/soletta/wiki/How-to-start).
+
+## Step 1
+
+### Flow Based Programming overview
+
+Soletta provides two API [3] for developers. A C API and a flow based
```

Comment:
don't use this kind of "[3]" for footnotes. If markdown lacks such, then just add within parenthesis